### PR TITLE
[datadog_integration_aws_available_namespaces] Create AWS Integrations V2 datasources `datadog_integration_aws_available_logs_services/namespaces`

### PR DESCRIPTION
**Pull request by ktmq**
Adds data sources using AWS Integrations V2 API: 
- `datadog_integration_aws_available_logs_services` to replace v1 `datadog_integration_aws_logs_services` data source
- `datadog_integration_aws_available_namespaces` to replace v1 `datadog_integration_aws_namespace_rules` data source

Adds docs on `datadog_integration_aws_account` fields to reference using these new data sources for allowed values.

### CODE REVIEW

**Review by HantingZhang2 — APPROVED** [review]

File: docs/resources/integration_aws_account.md [hunk 1]
```diff
@@ -139,8 +139,7 @@ Optional:
 Optional:
 
 - `lambdas` (List of String) List of Datadog Lambda Log Forwarder ARNs in your AWS account.
-- `sources` (List of String) List of service IDs set to enable automatic log collection. Discover the list of available services with the [Get list of AWS log ready services](https://docs.datadoghq.com/api/latest/aws-logs-integration/#get-list-of-aws-log-ready-services) endpoint.
-
+- `sources` (List of String) List of service IDs set to enable automatic log collection. Use [`datadog_integration_aws_available_logs_services` data source](https://registry.terraform.io/providers/DataDog/datadog/latest/docs/data-sources/integration_aws_available_logs_services) to get allowed values.
```
Comment (brett0000FF) [inline docs/resources/integration_aws_account.md:142]: Are these links supposed to work now? I'm getting "Page Not Found".

File: docs/data-sources/integration_aws_available_namespaces.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_aws_available_namespaces Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Use this data source to retrieve all available AWS Namespaces.
+---
+
+# datadog_integration_aws_available_namespaces (Data Source)
+
+Use this data source to retrieve all available AWS Namespaces.
```
Comment (brett0000FF) [inline docs/data-sources/integration_aws_available_namespaces.md:11]:
```suggestion
Use this data source to retrieve all available AWS namespaces.
```

File: docs/data-sources/integration_aws_available_namespaces.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `aws_namespaces` (List of String) List of Available AWS Namespaces.
```
Comment (brett0000FF) [inline docs/data-sources/integration_aws_available_namespaces.md:20]:
```suggestion
- `aws_namespaces` (List of String) List of available AWS namespaces.
```

**Review by brett0000FF — APPROVED** [review]
Approved with minor comment/suggestion. Thanks!

**Review by brett0000FF — APPROVED** [review]

**Review by saleelsaptarshi-dd — APPROVED** [review]

**Review by HantingZhang2 — APPROVED** [review]

**Pull request merged**